bulldozer/config_fetcher.go

```diff
@@ -25,6 +25,8 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+var commitMsgString = "==COMMIT_MSG=="
```

Comment:
Since this is only used for backwards compatibility, let's inline it in the one place it is used. That will be possible once the option is not a pointer (see other comments).